rules-reviewed/eap7/eap6/hibernate4-xml.windup.xml

```diff
@@ -11,8 +11,10 @@
             <addon id="org.jboss.windup.rules,windup-rules-java,2.4.0.Final" />
         </dependencies>
         <sourceTechnology id="hibernate" versionRange="[4,5)" />
+        <sourceTechnology id="eap" versionRange="[6,7)" />
         <targetTechnology id="hibernate5" versionRange="[5,)" />
         <targetTechnology id="eap7" versionRange="[7,)" />
+        <targetTechnology id="eap" versionRange="[7,)" />
```

Comment:
it is an experiment for that not working built-in types which implements org.hibernate.type.descriptor.sql.SqlTypeDescriptor. We have in EAP metadatas jars with APIs, so there is hibernate-core also for resolving the class hierarchy and to be activated that requires to have target=eap.